PSKoans/Koans/dbatools/AboutDbaDatabase.ps1

```diff
@@ -0,0 +1,87 @@
+#Requires -Modules dbatools
+using module PSKoans
+[Koan(Position = 1001)]
+param()
+<#
+    Get-DbaDatabase
+
+    The Get-DbaDatabase command gets SQL database information for each database
+    that is present on the target instance(s) of SQL Server by default. If the 
+    name of the database is provided, the command will return only the specific 
+    database.
+#>
+Describe 'Get-DbaDatabase' {
+
+    #region Mocks
```

Comment:
You might want to put a note in here for folks _not_ to mess with these or they'll be breaking everything haha. 😄

Reply:
Added a lil warning!